account_invoice_import_simple_pdf/models/account_invoice_import_simple_pdf_invoice_number.py

```diff
@@ -45,15 +45,17 @@ class AccountInvoiceImportSimplePdfInvoiceNumber(models.Model):
     @api.model
     def _string_type_sel(self):
         return [
-            ("fixed", "Fixed"),
-            ("year2", "Year on 2 digits"),
-            ("year4", "Year on 4 digits"),
-            ("month", "Month (2 digits)"),
-            ("digit", "Digit(s)"),
-            ("letter_upper", "Upper Letter"),
-            ("letter_lower", "Lower Letter"),
-            ("char", "Any Character"),
-            ("space", "Space"),
+            ("fixed", _("Fixed")),
```

Comment:
this is not needed for selection fields. The orm will flag these terms as translatable.

Reply:
@simahawk not "will flag " => "should theoretically flag"
In fact it's not working so I've added all missing terms.
Let me know if we can do better and force the orm to work